iFixit/react-commerce
# Move react-components into react-commerce monorepo

fixes #288 
 
## Changelog

- Move chakra theme into monorepo
- Move pagination component into monorepo
- Drop `react-components` dependency

## QA

1. Visit the Vercel PR preview (replace vercel.app with cominor.com) 
2. Navigate to `/Store/Parts`
3. Verify that the product list page still looks the same as main branch
